README.md

```diff
@@ -1,13 +1,61 @@
 [![Build Status](https://travis-ci.org/ofiwg/libfabric.svg?branch=master)](https://travis-ci.org/ofiwg/libfabric)
 
-fabtests
-========
+# fabtests
 
-Fabtests provides a set of examples that uses [libfabric](https://github.com/ofiwg/libfabric) - a high-performance fabric software library.
+Fabtests provides a set of examples that uses [libfabric](http://libfabric.org)
+-- a high-performance fabric software library.
 
-To make this directory, run:
+## Building fabtests
 
-    ./autogen.sh && ./configure && make && make install
+If you are building Fabtests from a developer Git clone, you must first run the
+autogen.sh script. This will invoke the GNU Autotools to bootstrap Fabtests'
+configuration and build mechanisms. If you are building Fabtests from an
+official distribution tarball from libfabric.org, there is no need to run
+autogen.sh; Fabtests distribution tarballs are already bootstrapped for you.
 
-Typically the autogen and configure steps only need be done the first
-time, unless configure.ac changes.
+Fabtests relies on being able to find an installed version of Libfabric. In
+some cases, Libfabric may be in default compiler / linker search paths, and you
+don't need to tell Fabtests where to find it. In other cases, you may need to
+tell Fabtests where to find the installed Libfabric's header and library files
+using the --with-libfabric=PATH option, described below.
+
+Configure ships with many built in options (see `./configure --help`). Some
+useful options are:
+
+```
```

Comment:
I changed it so that only the option lines were verbatim, I think I preferred the old look though. What do you think?